widget/index.js

```diff
@@ -165,7 +179,10 @@ var WidgetGenerator = yeoman.generators.Base.extend({
       this.template('nls/_strings.js', path.join(basePath, 'nls/strings.js'));
     }
     this.copy('images/icon.png', path.join(basePath, 'images/icon.png'));
-    this.template('_manifest.json', path.join(basePath, 'manifest.json'));
+    if(this.is2d)
```

Comment:
again, need to check if both 2d and 3d are supported, if so the logic here needs to be updated, and actually probably moved to w/in a single manifest.json template